### ISSUE TITLE
Read only end points

### ISSUE THREAD

**Issue by FreakTheMighty** [nestjsx/crud#30]
It it possible to hide some endpoints from a Crud controller? For example, if I wanted to expose a read only API, could I disable all end points accept for the `GET` endpoints?

**Comment by FreakTheMighty** [comment]
A pretty straightforward solution is to override the methods I want to hide and respond with a `404`. The remaining issue I'm seeing is the ability to remove those endpoints from swagger.

**Comment by pedroraft** [comment]
@FreakTheMighty i solved by using the following:
``` typescript
  @Override()
  @ApiOperation({ title: 'not implemented', deprecated: true })
  getMany(@Param() params, @Query() query: RestfulParamsDto) {
    return new NotFoundException();
  }
```
swagger puts the method in gray and scratches it as deprecated;

**Comment by zMotivat0r** [comment]
@FreakTheMighty @pedroraft the feature of excluding or choosing routes will be available in v3.0.0 soon. Cheers

**Comment by zMotivat0r** [comment]
Shipped with v3.0.0
